config.yaml

```diff
@@ -13,7 +11,7 @@ networks: # Configuration of the blockchain networks that the project is deploye
         events: # Configuration for each event emitted by this contract that the indexer will listen for
           - event: LiquidateBorrow #  Name of the event (must match the name in the ABI)
             requiredEntities: # An array of entities that need to loaded and made accessible within the handler function (an empty array indicates that no entities are required)
-              - name: LiquidatorAccount # The name of the required entity (must match an entity defined in `schema.graphql`)
-              - name: LiquidatedAccount
-              - name: TokenWon
-              - name: TokenLost
\ No newline at end of file
+              - name: Liquidatoraccount # The name of the required entity (must match an entity defined in `schema.graphql`)
```

Comment:
There is a feature in development that allows entities to be camelCase but currently entities are required to be lowercase. Sorry about this footgun.
It is mentioned [here](https://docs.envio.dev/docs/schema#other-design-tips) but is very subtle 🙈

Reply:
Changing the entities to not have camelcase is the biggest change